charts/pulsar/templates/presto-coordinator-statefulset.yaml

```diff
@@ -0,0 +1,166 @@
+          {{- if .Values.presto.coordinator.probe.readiness.enabled }}
+          readinessProbe:
+            httpGet:
+              path: /v1/cluster
+              port: {{ .Values.presto.coordinator.ports.http }}
+            initialDelaySeconds: {{ .Values.presto.coordinator.probe.readiness.initialDelaySeconds }}
+            periodSeconds: {{ .Values.presto.coordinator.probe.readiness.periodSeconds }}
+            failureThreshold: {{ .Values.presto.coordinator.probe.readiness.failureThreshold }}
+          {{- end }}
+          {{- if .Values.presto.coordinator.resources }}
+          resources:
+{{ toYaml .Values.presto.coordinator.resources | indent 12 }}
+          {{- end }}
+          command: ["sh", "-c"]
+          args:
+            - >-
+              bin/pulsar sql-worker run \
+                --etc-dir={{ template "pulsar.home" . }}/conf/presto \
+                --data-dir={{ template "pulsar.home" . }}/data;
+          volumeMounts:
+            - mountPath: {{ template "pulsar.home" . }}/conf/presto/node.properties
+              name: config-volume
+              subPath: node.properties
+            - mountPath: {{ template "pulsar.home" . }}/conf/presto/log.properties
+              name: config-volume
+              subPath: log.properties
+            - mountPath: {{ template "pulsar.home" . }}/conf/presto/jvm.config
+              name: config-volume
+              subPath: jvm.config
+            - mountPath: {{ template "pulsar.home" . }}/conf/presto/config.properties
+              name: config-volume
+              subPath: config.properties
+            - mountPath: {{ template "pulsar.home" . }}/conf/presto/catalog/pulsar.properties
+              name: config-volume
+              subPath: pulsar.properties
+            {{- if .Values.auth.authentication.enabled }}
+            {{- if eq .Values.auth.authentication.provider "jwt" }}
+            - mountPath: "/pulsar/tokens"
+              name: client-token
+              readOnly: true
+            {{- end }}
+            {{- end }}
+            {{- if and .Values.tls.enabled .Values.tls.broker.enabled }}
+            - mountPath: "/pulsar/certs/ca"
+              name: ca
+              readOnly: true
+            {{- end}}
+          ports:
+            - name: http-coord
+              containerPort: {{ .Values.presto.coordinator.ports.http }}
+              protocol: TCP
+      volumes:
+        {{- if .Values.auth.authentication.enabled }}
+        {{- if eq .Values.auth.authentication.provider "jwt" }}
+        - name: client-token
+          secret:
+            secretName: "{{ .Release.Name }}-token-{{ .Values.auth.superUsers.client }}"
+            items:
+              - key: TOKEN
+                path: client/token
+        {{- end}}
+        {{- end}}
+        {{- if and .Values.tls.enabled .Values.tls.broker.enabled }}
+        - name: ca
+          secret:
+            secretName: "{{ template "pulsar.fullname" . }}-ca-tls"
```

Comment:
I'm not sure should the secret name need to use a release name to be as a prefix of the secret name?